src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisController.java

```diff
@@ -229,130 +230,215 @@ public String getUserAnalysisOutputsPage(Model model) {
 	 *
 	 * @param submissionId the ID of the submission
 	 * @param model        Model for the view
+	 * @param principal    Principal {@link User}
 	 * @param locale       User's locale
 	 * @return name of the details page view
 	 */
+
 	@RequestMapping(value = "/{submissionId}", produces = MediaType.TEXT_HTML_VALUE)
-	public String getDetailsPage(@PathVariable Long submissionId, Model model, Locale locale) {
+	public String getDetailsPage(@PathVariable Long submissionId, Model model, final Principal principal,
+			Locale locale) {
```

Comment:
`locale` is no longer used.

Reply:
Fixed in 9c4e640d5c452bf1455f9b93a1b1499a418283e6